inference-engine/src/plugin_api/cpp_interfaces/impl/ie_infer_async_request_internal.hpp

```diff
@@ -13,6 +13,10 @@
 
 namespace InferenceEngine {
 
+#if defined(_MSC_VER)
+#pragma warning(disable : 4250)
+#endif
```

Comment:
should not we use warning push / pop since it's a header file

Reply:
This is basically warning about virtual inheritance being used (the only way to "fix" it is to rewrite code without virtual inheritance). Since we use it anyway and this header is intended for plugin internals I think it is fine to just turn it off. Also GCC/Clang do not warn about this at all.